docs/messaging/messaging-spans.md

```diff
@@ -479,11 +484,11 @@ flowchart LR;
   end
   subgraph CONSUMER1
   direction TB
-  R1[Span Deliver A 1]
+  R1[Span Process A 1]
```

Comment:
I think this might have been discussed before, but what is the reasoning that these are links instead of parent/child relationships? My understanding was that links are needed for scenarios where a span might have multiple parents but this doesn't seem to be the case. Should we offer that either (links or parent/child) can be used?

Reply:
We require links to be present, as that's the only reliable correlation mechanism that can work consistently across all messaging scenarios.

Parent/child relationships can be added in addition to links where feasible, but aren't required by the conventions. For the sake of keeping examples minimal, I left it out there.

Reply:
Yeah that sounds good. Thanks for the clarification.